docs/docs/ScalaUserGuide/build-bigdl-core.md

```diff
@@ -125,6 +135,9 @@ can use `brew`.
 
 We use maven profile to control the build process. For different platforms has different profiles.
 
+$source ${Parallel_Studio_XE_HOME}/compilers_and_libraries/linux/bin/compilervars.sh
```

Comment:
why this line ?

Reply:
without sourcing ICC environment variable from Parallel Studio installation, I would see an compilation error due to missing icc command.

Reply:
but this is for your environment specific right ? we don't want this from framework's perspective

Reply:
Hi Jerry.  the only difference among my environment and other users' environment should be where they install Parallel Studio XE. therefore, I make the parallel studio installation path as a variable {Parallel_Studio_XE_HOME}, and users need to fill path themselves.  
we can add some explanation there. We could also remove this line, and let users handle it by themselves. what is your preference?

Reply:
My understanding is  Parallel Studio is not required for compiling bigdl core right ?  I would recommend to remove it to avoid confusion

Reply:
My understanding is  Parallel Studio is not required for compiling bigdl core right ?  I would recommend to remove it to avoid confusion

Reply:
sure. will do.

Reply:
remove the line about setting ICC env variable